chain/chain/Cargo.toml

```diff
@@ -50,8 +50,6 @@ no_cache = ["near-store/no_cache"]
 protocol_feature_flat_state = ["near-store/protocol_feature_flat_state"]
 protocol_feature_reject_blocks_with_outdated_protocol_version = ["near-primitives/protocol_feature_reject_blocks_with_outdated_protocol_version"]
 
-shardnet = ["protocol_feature_reject_blocks_with_outdated_protocol_version"]
```

Comment:
Do we want to get rid of the feature here completely? Since we might want to bring up shardnet again in Q1/Q2 2023.

Reply:
yeah - I'd clean it up. When we're ready in Q2, we'll re-add it again (as currently there was a bunch of stuff left behind that caused the node to assert -- that is the issue that @wacban hit ).